app/user/client/client.form.pug

```diff
@@ -51,6 +51,15 @@ hr
         ng-disabled="!clientForm.billing.integration.selected"
       )
 
++label(
```

Comment:
we should combine all of the checkbox options (this and "Ignore auto suspend/terminate") under a single "Privileges" label with the multiple checkboxes.

Also, for frontend code, it really helps to provide a screenshot of what the solution looks like so reviewers don't have to guess about what it looks like or pull it up on their machine.